Khan/perseus
# Image widgets: Centre everywhere except hints

## Summary:

This PR updates image rendering so that `image` widgets (but _not_ bare Markdown image links) are rendered with new alignments:

  * In articles: centred
  * In exercises: centred
  * In hints: left

Note: The Image Alignment [1Pager](https://khanacademy.atlassian.net/wiki/spaces/LC/pages/2607218692/Allow+for+both+center-+and+left-aligned+images+in+articles+and+exercises+1P) requests more control than this PR provides so there may be some future work.  

However, LEMS-1798 lays out the requirements as I've stated them so this at least gets us to more aesthetically pleasing image rendering across the board. 

Issue: LEMS-1798

## Test plan:

Test on both Desktop and Mobile...

  * Navigate to the [Murals and Public Art in 1930s Rome](https://www.khanacademy.org/humanities/art-1010/german-art-between-the-wars/xdc974a79:nazi-fascist-visual-culture/a/murals-and-public-art-in-1930s-rome) article. Note that all images are either "full bleed" or centred horizontally if they are not the width of the article content. 

  * Navigate to the [Apply: ionic bonds and compounds](https://www.khanacademy.org/science/hs-chemistry/x2613d8165d88df5e:chemical-bonding/x2613d8165d88df5e:ionic-bonds/e/apply-ionic-bonds-and-compounds) exercise. Note that hints that contain an image are left-aligned (such as in the question "Strontium chloride is an ionic compound. It is used in fireworks to produce red light.").

  * Navigate to the [German art between the wars](https://www.khanacademy.org/humanities/art-1010/german-art-between-the-wars/test-german-art-between-wars/e/german-art-between-the-wars) exercise. Note that the images in the exercise content are centred.

## Examples from production

| device | url | screenshot |
| --- | --- | --- |
| Desktop | [German art between the wars](https://www.khanacademy.org/humanities/art-1010/german-art-between-the-wars/test-german-art-between-wars/e/german-art-between-the-wars) | <img width="500" alt="Screenshot 2024-03-20 at 1 29 08 PM" src="https://github.com/Khan/perseus/assets/77138/f1e02904-b36d-4407-b3c6-a6306993df99"> |
| Phone | [German art between the wars](https://www.khanacademy.org/humanities/art-1010/german-art-between-the-wars/test-german-art-between-wars/e/german-art-between-the-wars) | <img width="500" alt="image" src="https://github.com/Khan/perseus/assets/77138/8c27f60f-002b-4287-beca-56806b119be2"> | 
| Desktop | [Murals and Public Art in 1930s Rome](https://www.khanacademy.dev/humanities/art-1010/german-art-between-the-wars/xdc974a79:nazi-fascist-visual-culture/a/murals-and-public-art-in-1930s-rome) | <img width="500" alt="Screenshot 2024-03-20 at 1 57 08 PM" src="https://github.com/Khan/perseus/assets/77138/6fe741b0-d782-4410-997e-e6213403f21c"> |
| Mobile | [Murals and Public Art in 1930s Rome](https://www.khanacademy.dev/humanities/art-1010/german-art-between-the-wars/xdc974a79:nazi-fascist-visual-culture/a/murals-and-public-art-in-1930s-rome) | <img width="500" alt="image" src="https://github.com/Khan/perseus/assets/77138/391673d3-b7d7-42b6-9a44-cf1034776e56"> |
| Desktop | [Apply: ionic bonds and compounds](https://www.khanacademy.dev/science/hs-chemistry/x2613d8165d88df5e:chemical-bonding/x2613d8165d88df5e:ionic-bonds/e/apply-ionic-bonds-and-compounds) |  |
| Mobile | [Apply: ionic bonds and compounds](https://www.khanacademy.dev/science/hs-chemistry/x2613d8165d88df5e:chemical-bonding/x2613d8165d88df5e:ionic-bonds/e/apply-ionic-bonds-and-compounds) | <img width="500" alt="image" src="https://github.com/Khan/perseus/assets/77138/1c4e1e31-eaa2-4d3e-89e9-b3c0d1322d89"> |
